content/migrations/react.md

```diff
@@ -0,0 +1,278 @@
+# Migrating From React
+Choo borrows heavily from React. For many of us, it was the first experience we
+had with declarative rendering, and separation of events and data.
+
+However, there are a good amount of differences. At its core Choo is
+"declarative rendering" + "shared event emitter" + "router". Once you understand
+how these pieces fit together, you'll quickly be able to create an intuition for
+how Choo operates.
+
+## Creating New Projects
+React recommends using
+[create-react-app](https://github.com/facebookincubator/create-react-app) to
+create new project. Choo has a similar project called
+[create-choo-app](https://github.com/choojs/create-choo-app/). The output is
+different, but the process should feel familiar coming from React.
+
+## Declarative Rendering
+Both React and Choo have a system that allows you to define of "this is what the
+DOM should look like". React uses JSX, and Choo uses Template String Literals.
```

Comment:
> React uses JSX

This is a huge difference, and the main reason why I've never wanted to try React. Should we talk a bit more about the non-standard jsx? I mean how it forces you to use some specific environment config. Choo on the other side expect native DOM, we could event replace `choo/html` and use any other DOM librarie. IMO, this difference is huge.

Reply:
Yes, although I don't think this guide should serve as a list of things that Choo has over React :P